docker_run.sh

```diff
@@ -1 +1 @@
-docker run -d -v $("pwd")/volume-blueocean:/var/jenkins_home -p 8085:8080 -p 50000:50000 --name=jenkins-blueocean -i jenkinsci/blueocean:1.3.3
\ No newline at end of file
+docker run -d -v $("pwd")/volume-blueocean:/var/jenkins_home -p 8081:8080 -p 49001:49001 --env JENKINS_SLAVE_AGENT_PORT=49001 --name=jenkins-blueocean  -i jenkinsci/blueocean
```

Comment:
Why do we need to change `agent_port` and `jenkins_web_port` ? 
And why do we need `JENKINS_SLAVE_AGENT_PORT` ?

Reply:
because in RP they don't open port 50000 but 49001 for us, and you need to route the env setup for jenkins via this way as well as open i/o port 49001:49001 for our container